src/Requirements.md

```diff
@@ -28,10 +28,28 @@
 
 @anchor R14 14.The game shall enable users to walk at different speeds on blocks with different textures.
 
+<<<<<<< HEAD
```

Comment:
merge conflict need to fix

Reply:
upload one you can directly replace yours.

Reply:
Sorry, I'm still not super used to GitHub. Do I just delete the requirements.md from my branch since it's already in master?